qa/zcash/performance-measurements.sh

```diff
@@ -141,6 +167,10 @@ case "$1" in
             incnotewitnesses)
                 zcash_rpc zcbenchmark incnotewitnesses 1 "${@:3}"
                 ;;
+            connectblockslow)
+                extract_benchmark_data
+                zcash_rpc zcbenchmark connectblockslow 1
```

Comment:
That parameter is the number of times to run the benchmark (and therefore the number of timing measurements the call will return). For timing benchmarks, we want several data points so that statistics can be drawn (which is done automatically by the CI system). For memory benchmarks and valgrind, we only run once because the instrumentation greatly slows the execution time.